Scripts/import-logs.ps1

```diff
@@ -0,0 +1,110 @@
+#########powershell script to upload log files into s3 bucket###########
+
+#s3Bucket path to save the log files
+param (
+    [string] $version,
+    [string] $ReleaseId
+)
+$Lansaversion = $version
+$PipelineReleaseId = $ReleaseId
+$s3BasePath = 'lansa-us-east-1/db-regression-test/logs' 
+$region = 'us-east-1'
+ 
+#Total Summary for all Dbs
+$totalSummaryFile = 'C:\Program Files (x86)\LANSA\Verifier_Total_Summary.txt'
+$totalSummaryPath = Join-Path $Lansaversion $PipelineReleaseId
+$totalSummaryKey = $totalSummaryPath + '/Verifier_Total_Summary.txt'
+Write-S3Object -BucketName $s3BasePath -Key $totalSummaryKey -Content $totalSummaryFile 
```

Comment:
You need to use log messages. This is not possible to diagnose issues right now.
Use Write-Host as described in Teams.
So for example, here you need to log `$totalSummaryKey ` and `$totalSummaryFile`.
Also add `| Out-Default | Write-Host` to the end of the `Write-S3Object` line.

With log messages you could simply provide those to me to answer the questions in the Issue about SPECIFIC paths.